internal/datastore/postgres/options.go

```diff
@@ -38,7 +43,9 @@ type Option func(*postgresOptions)
 
 func generateConfig(options []Option) (postgresOptions, error) {
 	computed := postgresOptions{
-		gcWindow:                  24 * time.Hour,
+		gcWindow:                  defaultGarbageCollectionWindow,
+		gcInterval:                defaultGarbageCollectionInterval,
```

Comment:
I'm open to suggestions :) I think `gcMaxOperationTime` is pretty explanatory. As for `gcWindow` vs `gcInterval`, maybe `gcRunFrequency`?

Reply:
snapshotGC vs transactionGC?

Reply:
Its not those, though. `gcInterval` is how often is runs. `gcWindow` is the duration until things are marked as garbage and thereafter collectable

Reply:
from a user's perspecitve, `gcWindow` affects how old of a revision they're allowed to use to query (it's really not related to GC at all in the crdb datastore for example). It's a setting that directly affects end users

gcInterval and gcMaxOperationTime are more ops-facing flags, they affect how garbage is managed over time. 

maybe this is picking too many nits, they just seem like a different class of thing to me. I wouldn't block the PR on this, but I do think there's probably a clearer way to name this.

Reply:
`gcWindow` only does that in CRDB and memdb because they have different means of collecting garbage (or not at all). After this change, it does directly affect postgres, because anything older than that window will (eventually) be deleted